docs/reference/query-languages/esql/esql-commands.md

```diff
@@ -191,6 +192,69 @@ SHOW INFO
 | 8.13.0 | 2024-02-23T10:04:18.123117961Z | 04ba8c8db2507501c88f215e475de7b0798cb3b3 |
 
 
+## `CHANGE_POINT` [esql-change-point]
```

Comment:
Since the merger of https://github.com/elastic/elasticsearch/pull/126279, this now needs to be done differently. You need to move this entire section into a file called `_snippets/commands/layout/change_point.md` and include that from the file `commands/processing_commands.md`. Then you need to edit the change_point.md file to include the right tests from `_snippets/commands/examples/*`.